test/runnable/depsprot.sh

```diff
@@ -3,23 +3,19 @@
 name=`basename $0 .sh`
 dir=${RESULTS_DIR}/runnable
 dmddir=${RESULTS_DIR}${SEP}runnable
-output_file=${dir}/${name}.sh.out
```

Comment:
Looks like this script wanted to print its output only on failure? Is its output overly verbose, or something?

Reply:
All output is already printed on a failure, so this is just redundant.